docs/platform_concepts/mobile/chatbot/flutter.md

```diff
@@ -231,6 +231,14 @@ Bot can be programmatically closed using `closeBot()` function
 YmChat.closeBot();
 ```
 
+## Reload bot
+
+Bot can be reloaded with the same config bot started with, using the `reloadBot()` function.
```

Comment:
If you want to reload the bot with same configs you have started it, you can do so by calling reloadBot() function.
This will reload the bot without closing and reopening it.